repo/linux/tzungbi-chrome-platform

```diff
@@ -2,3 +2,4 @@ url: https://git.kernel.org/pub/scm/linux/kernel/git/tzungbi/chrome-platform.git
 owner: Tzung-Bi Shih <tzungbi@kernel.org>
 mail_cc: Tzung-Bi Shih <tzungbi@kernel.org>
 notify_build_success_branch: .*
+private_report_branch: .*
```

Comment:
thanks, and also help update mail_cc to mail_to if using private_report_branch. Sorry if this is confusing, it is what is implemented now.

Reply:
Done: replace mail_cc to mail_to.